src/robot_parameters.cc

```diff
@@ -133,6 +133,12 @@ bool RobotParameters::UpdateUrdf(const std::string& new_urdf_file_name) {
   return true;
 }
 
+// local function only used by loadYamlParameters below
+void ConstructPaths(RobotParameters& params) {
```

Comment:
This function seems to more than `ConstructPaths`, no?

Reply:
kept the old name and code. i don't know we even needed it. i removed it now.